analyzer/src/funTest/kotlin/managers/SpdxDocumentFileFunTest.kt

```diff
@@ -40,9 +40,9 @@ import org.ossreviewtoolkit.utils.test.DEFAULT_REPOSITORY_CONFIGURATION
 import org.ossreviewtoolkit.utils.test.USER_DIR
 import org.ossreviewtoolkit.utils.test.patchExpectedResult
 
-class SpdxDocumentFileFunTest : StringSpec() {
-    init {
-        "Project dependencies are detected correctly" {
+class SpdxDocumentFileFunTest : WordSpec({
+    "Dependencies" should {
```

Comment:
How about "resolveDependencies()" should { " ...."
to make clear that resolveDependencies() is being tested. Like below with "mapDefinitionFiles()"